lib/ts/recipe/emailpassword/types.ts

```diff
@@ -391,6 +392,7 @@ export type EnterEmailStatus = "READY" | "SENT";
 export type SubmitNewPasswordStatus = "READY" | "SUCCESS";
 
 export type FormBaseProps<T> = {
+    formDataSupertokens?: string;
```

Comment:
what is this for?

Reply:
It adds a `data-supertokens` prop to the form element. Used internally for testing.